helm/v1beta1/0000_10-helm-chart-repository.crd.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helmchartrepositories.helm.openshift.io
+spec:
+  scope: Cluster
+  preserveUnknownFields: false
+  group: helm.openshift.io
+  names:
+    kind: HelmChartRepository
+    listKind: HelmChartRepositoryList
+    plural: helmchartrepositories
+    singular: helmchartrepository
+    shortNames:
+    - hcr
```

Comment:
I doubt this will be commonly retreived.  Avoid claiming shortnames for rarely (by end user/clusteradmins) `oc` accessed resources.

Reply:
ok.